### PR TITLE
Updating Product name with Azure Arc

### PR DESCRIPTION
**Pull request by nasc17**
Updating product name of _Azure Arc enabled PostgreSQL Hyperscale_ to **Azure Arc-enabled PostgreSQL Hyperscale**

### CODE REVIEW

File: resources/localization/LCL/de/azurecore.xlf.lcl [hunk 1]
```diff
@@ -833,7 +833,7 @@
       </Item>
       <Item ItemId=";extensions/azurecore/dist/localizedConstants.azurecore.azureArcPostgres" ItemType="0" PsrId="308" Leaf="true">
         <Str Cat="Text">
-          <Val><![CDATA[Azure Arc enabled PostgreSQL Hyperscale]]></Val>
+          <Val><![CDATA[Azure Arc-enabled PostgreSQL Hyperscale]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
             <Val><![CDATA[PostgreSQL Hyperscale mit Azure Arc-Aktivierung]]></Val>
```
Comment (Charles-Gagnon) [inline resources/localization/LCL/de/azurecore.xlf.lcl:838]: Don't update these files - they're generated by the loc team and are for mapping the string IDs to their localized values. But the English version is just for reference - the actual value you'd want to update is the "val" on line 838.

So what you should be doing is just updating the source localize string (in this case `azurecore.azureArcPostgres`) and then the loc team should update it on their next pass.

@smartguest @kburtram Do we have a set schedule for when they do this or is it just whenever their schedule allows?

File: resources/xlf/de/azurecore.de.xlf [hunk 2]
```diff
@@ -379,7 +379,7 @@
         <target state="translated">SQL Server – Azure Arc</target>
       </trans-unit>
       <trans-unit id="azurecore.azureArcPostgres">
-        <source xml:lang="en">Azure Arc enabled PostgreSQL Hyperscale</source>
+        <source xml:lang="en">Azure Arc-enabled PostgreSQL Hyperscale</source>
```
Comment (Charles-Gagnon) [inline resources/xlf/de/azurecore.de.xlf:382]: Same here for the .xlf files - you shouldn't be updating these manually

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: resources/localization/LCL/de/azurecore.xlf.lcl (hunk 1 above)
Comment (smartguest) [reply resources/localization/LCL/de/azurecore.xlf.lcl:838]: the LCL files are generated whenever we check in new resources generated from XLFs created from the extensions, my new function in my pull request is designed to create these files and whenever I check them in, LCL files will be generated within a few days, depending on how fast the loc team gets to it.

**Comment by smartguest** [comment]
@nasc17 please remove the changes in XLF and LCL files, they will be generated when my XLF generation function is added to ADS and the changes will be included with the next XLF refresh.

**Comment by nasc17** [comment]
Made new PR: https://github.com/microsoft/azuredatastudio/pull/15845

File: resources/localization/LCL/de/azurecore.xlf.lcl (hunk 1 above)
Comment (Charles-Gagnon) [reply resources/localization/LCL/de/azurecore.xlf.lcl:838]: Do we have this process documented anywhere? Are they just checking the repo every couple of days or is there some trigger that tells them there's updated strings to localize?

Comment (smartguest) [reply resources/localization/LCL/de/azurecore.xlf.lcl:838]: According to @khoiph1, what happens is that:

When an updated English XLF is checked into the repo, it triggers a new pipeline build at Runs for azuredatastudio-CI once a day.

The loc team's localization tool scans the build and picks up changes in the artifact. 

Later, once the translations are completed, the LCL is updated by a bot, and it creates a new build in the above pipeline, inside the drop are the updated localized XLF files to be committed back into ADS.

**Pull request closed without merging**